nixos/modules/system/boot/loader/grub/grub.nix

```diff
@@ -57,6 +57,7 @@ let
       storePath = config.boot.loader.grub.storePath;
       bootloaderId = if args.efiBootloaderId == null then "${config.system.nixos.distroName}${efiSysMountPoint'}" else args.efiBootloaderId;
       timeout = if config.boot.loader.timeout == null then -1 else config.boot.loader.timeout;
+      timeoutStyle = cfg.timeoutStyle;
```

Comment:
Let's reuse the `inherit (cfg)` we already have below instead?

Reply:
https://github.com/NixOS/nixpkgs/pull/220263/files#diff-713d422d999999a630d109add2653608eb7eacade640f61f781ec521c47afb7eR91